views/js/front/pudo-search.js

```diff
@@ -228,7 +228,7 @@ function inArray(needle, haystack) {
 
 function togglePhoneRequiredField(defaultCarrier) {
     let deliveryMethod = $('.custom-radio > input');
-    let currentCarrier = defaultCarrier.props('defaultValue');
+    let currentCarrier = defaultCarrier.val();
```

Comment:
I think this is for the defaulValue (the first value for object), val() returns current value.

So i don't think this is accurate to do, unless it works by switching carriers and nothing bad occurs :]

Reply:
Nothing bad occurs it extracts carrier ID witch I need to compare if it's dpd carrier to disable the phone field required or not.

Reply:
I guess that's OK